tests/test_exchange_rate.py

```diff
@@ -0,0 +1,66 @@
+from datetime import date, timedelta
+from capgains.exchange_rate import ExchangeRate
+import pytest
+from click import ClickException
+import requests_mock as rm
+
+
+@pytest.fixture(scope='function')
+def USD_exchange_rates(requests_mock):
+    requests_mock.get(rm.ANY,
+                      json={
+                        "observations": [
+                            {
+                                "d": "2020-05-22",
+                                "FXUSDCAD": {
+                                    "v": "1.2"
+                                }
+                            }
+                        ]})
+    return ExchangeRate('USD',
+                        ExchangeRate.min_date,
+                        date.today())

```

Comment:
Why do you need to return this object? Isn't it enough to just create the request_mock call?

Reply:
Not issue